lib/cisco_node_utils/interface.rb

```diff
@@ -903,6 +914,444 @@ def switchport_trunk_native_vlan=(val)
       end
     end
 
+    # Interface private vlan configuration properties
+    # private vlan mode (host, promisc, trunk promisc , trunk sec)
+    # private vlan mapping
+    # private vlan native vlan
+    # private vlan allow vlan
+
+    def cli_error_check(result)
```

Comment:
Let's not duplicate this here. Better to either:

(a) Move it to cisco_cmn_utils.rb
-or-
(b) Make the cli_error_check in vlan.rb a class method

My vote is (a).